### PR TITLE
[Translation] Docker Context article to ID

### PR DESCRIPTION
**Pull request by paramithatm**
Translated article: working-with-remote-docker-using-docker-context.md

### CODE REVIEW

File: content/id/blogs/working-with-remote-docker-using-docker-context.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+my-remote-docker-machine
+Current context is now "my-remote-docker-machine"
+$ docker context ls
+NAME                         DESCRIPTION                               DOCKER ENDPOINT               KUBERNETES ENDPOINT   ORCHESTRATOR
+default                      Current DOCKER_HOST based configuration   unix:///var/run/docker.sock                         swarm
+my-remote-docker-machine *                                             ssh://username@host
+
+```
+
+See the `*` is moved from `default` to `my-remote-docker-machine`. Now you can use your docker command without `--context` flag.
```
[the remark this answers is not in the document]
Comment (paramithatm) [reply content/id/blogs/working-with-remote-docker-using-docker-context.md]: updated @clavinjune

File: content/id/blogs/working-with-remote-docker-using-docker-context.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
+---
+title: "Bekerja dengan Remote Docker menggunakan Docker Context"
+date: 2021-12-02T15:36:25+07:00
```
Comment (clavinjune) [inline content/id/blogs/working-with-remote-docker-using-docker-context.md:3]: update to today's date

Comment (paramithatm) [reply content/id/blogs/working-with-remote-docker-using-docker-context.md]: updated

**Review by clavinjune — APPROVED** [review]
LGTM

**Pull request merged**